test/addons/dlopen-ping-pong/binding.cc

```diff
@@ -26,6 +26,7 @@ static ping ping_func;
 void LoadLibrary(const FunctionCallbackInfo<Value>& args) {
   const String::Utf8Value filename(args.GetIsolate(), args[0]);
   void* handle = dlopen(*filename, RTLD_LAZY);
+  if (handle == nullptr) fprintf(stderr, "%s\n", dlerror());
```

Comment:
Will https://github.com/cpplint/cpplint bo OK with this all on one line?

Reply:
No idea, but tools/cpplint.py is fine with it.

Reply:
Cool.  Ship it.